pkg/services/alerting/notifiers/pushover.go

```diff
@@ -142,6 +143,9 @@ func (this *PushoverNotifier) Notify(evalContext *alerting.EvalContext) error {
 	if evalContext.ImagePublicUrl != "" {
 		message += fmt.Sprintf("\n<a href=\"%s\">Show graph image</a>", evalContext.ImagePublicUrl)
 	}
+	if message == "" {
+		message = "Nothing to see here! (Set a notification message to replace this text.)"
```

Comment:
May be just `notification message missing` ?

Reply:
Feel free to change that to your liking.